client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -106,7 +106,7 @@
         <input
           type="checkbox"
           data-cy="adminLayerListItem:toggleDefaultVisibility"
-          :disabled="'' !== layer.attributes.visibilityGroupId || (true === isChildOfCategoryThatAppearsAsLayer)"
+          :disabled="null !== layer.attributes.visibilityGroupId || (true === isChildOfCategoryThatAppearsAsLayer)"
```

Comment:
may it would be even more resiliant like so:
```suggestion
          :disabled="layer.attributes.visibilityGroupId || (true === isChildOfCategoryThatAppearsAsLayer)"
```